IRCT-API/src/main/java/edu/harvard/hms/dbmi/bd2k/irct/controller/QueryController.java

```diff
@@ -306,12 +306,14 @@ private void validateWhereClause(Resource resource, Entity field,
 			throw new QueryException(
 					"Predicate is not supported by the resource");
 		}
+
 		// Does the predicate support the entity?
-		if ((!predicate.getDataTypes().isEmpty())
+		/*if ((!predicate.getDataTypes().isEmpty())
```

Comment:
This won't be a problem anymore?

Reply:
Hm... never seen any query is using predicate with Entity style before, so this change should be fine...

Reply:
I preferred not changing it

Reply:
Will change it back and test it